src/sub/messages.py

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import json
+
+import boto3
+
+from abc import ABC
+
+from sub.shared.cfg import CFG
+from shared.log import get_logger
+
+from botocore.exceptions import ClientError
```

Comment:
group `from imports` of external code _above_ our code imports

Reply:
@scottidler updated, please review.